tools/sof-tplgreader.py

```diff
@@ -172,8 +172,11 @@ def _filterOutput(self, target_lst, filter_dict, bIn):
                 if 'any' in value or value == ['']:
                     check = True if key in line.keys() else False
                 else:
-                    # match for 'keyword'/'keyword [0-9]' target line
-                    check = len ([em for em in value if re.match(em + '$|' + em + '[^a-zA-Z]', str(line[key]), re.I)]) > 0
```

Comment:
Some of Pylint warnings are making sense.

```
tools/sof-tplgreader.py:3:0: W0611: Unused import subprocess (unused-import)
tools/sof-tplgreader.py:4:0: W0611: Unused import os (unused-import)
tools/sof-tplgreader.py:5:0: W0611: Unused import re (unused-import)
```

Reply:
pylint warnings fixed in no functional change PR #714, please help review